index.js

```diff
@@ -46,50 +46,8 @@ if (!isProduction) {
 app.use("/api-docs", swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 app.use("/api/v1/", routers);
 
-// catch 404 and forward to error handler
-app.use(next => {
-  const err = new Error("Not Found");
-  err.status = 404;
-  next(err);
-});
-
-// error handlers
-
-// development error handler
-// will print stacktrace
-if (!isProduction) {
-  app.use((err, res) => {
-    // eslint-disable-next-line no-console
-    console.log(err.stack);
-
-    res.status(err.status || 500);
-
-    res.json({
-      errors: {
-        message: err.message,
-        error: err
-      }
-    });
-  });
-}
-
-// production error handler
-// no stacktraces leaked to user
-app.use((err, res) => {
-  res.status(err.status || 500);
-  res.json({
-    errors: {
-      message: err.message,
-      error: {}
-    }
-  });
-});
-
-// finally, let's start our server...
 const server = app.listen(process.env.PORT || 3000, () => {
-  // eslint-disable-next-line no-console
   console.log(`Listening on port ${server.address().port}`);
```

Comment:
Can't this `console.log()` be removed?

Reply:
@YvesIraguha we decided to let it there in the meanwhile.